docs/en/test_and_ci/integration_testing_ros1_mavros.md

```diff
@@ -0,0 +1,155 @@
+# Integration Testing using ROS 1
+
+This topic explains how to run (and extend) PX4's ROS (1) and MAVROS -based integration tests.
+
+:::warning This test framework is deprecated.
+It should be used only for new test cases that _require_ ROS 1.
+
+[MAVSDK Integration Testing](../test_and_ci/integration_testing_mavsdk.md) is preferred when writing new tests.
+:::
+
+:::note
+All PX4 integration tests are executed automatically by our [Continuous Integration](../test_and_ci/continous_integration.md) system.
+:::
+
+## Prerequisites
+
+- [jMAVSim Simulator](../sim_jmavsim/index.md)
+- [Gazebo Classic Simulator](../sim_gazebo_classic/index.md)
+- [ROS and MAVROS](../simulation/ros_interface.md)
```

Comment:
I am removing jMAVSim since it is now community supported, and maintained unreliably.

```suggestion
- [Gazebo Classic Simulator](../sim_gazebo_classic/index.md)
- [ROS and MAVROS](../simulation/ros_interface.md)
```